src/Products/Signature/Controllers/SignatureApiController.cs

```diff
@@ -1124,26 +1151,24 @@ private SignedDocumentEntity SignDocument(string documentGuid, string password,
         /// <param name="password">string</param>
         /// <param name="signsCollection">SignatureOptionsCollection</param>
         /// <returns></returns>
-        private Stream SignDocumentStream(string documentGuid, string password, SignatureOptionsCollection signsCollection)
+        private Stream SignDocumentStream(string documentGuid, string password, List<SignOptions> signsCollection)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'SignDocumentStream' a static method.](https://app.codacy.com/gh/groupdocs-signature/GroupDocs.Signature-for-.NET-MVC/pullRequest?prid=5325146)